kubefed/installing-kubefed.adoc

```diff
@@ -0,0 +1,31 @@
+[id="installing-kubefed"]
+= Installing OpenShift KubeFed
+include::modules/common-attributes.adoc[]
+include::modules/kubefed-document-attributes.adoc[]
+:context: installing-kubefed
+
+To get started with {KubeFedProductName}, you must create the `openshift-federation-system` namespace, install the Operator in this namespace, and then create the required custom resources (CRs) for your deployment.
+
+NOTE: {KubeFedProductName} can be deployed as either a namespace-scoped or a cluster-scoped deployment, but not both, as this will cause conflicts when both deployments attempt to manage the same set of resources.
+
+== Creating the namespace
+
+You must create the `openshift-federation-system` namespace before installing any other {KubeFedProductName} components by using the following command:
```

Comment:
From mod docs guide: "For single-step procedures, use an unnumbered bullet instead of a numbered list."